pkgs/development/python-modules/acstore/default.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+  pyyaml,
+  setuptools,
+  ...
+}:
+buildPythonPackage rec {
+  pname = "acstore";
+  version = "20240407";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-yubHDEZ5nwltQW8sLEAhgyaXI0svHCS3a7Mewi6cvpg=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [ pyyaml ];
+
+  disabled = pythonOlder "3.8";
+
```

Comment:
Could you please add `pytestCheckHook` to `nativeCheckInputs` and enable tests?

Reply:
Certainly - apologies, I had assumed that buildPythonPackage without a `doCheck = false;` achieved this, can do :+1: 

A bit of guidance would be great on the first rebase instance I'm tackling; `acstore` returns an error of:
```
/build/acstore-20240407/tests/test_lib.py:21: PytestCollectionWarning: cannot collect test class 'TestAttributeContainer' because it has a __init__ constructor (from: tests/test_lib.py)
```

I assume there's a few ways to approach this - is there a preferred way in nixpkgs? My naive assumptions of common path would be (in preference order):
* remove the __init__.py within test folders
* remove the failing test (though not applicable here)
* disable tests

Assuming removal of the __init__ is suggested - is there a preferred hook or way to do this?